docs/pages/getting-started/3-configure-your-node.rst

```diff
@@ -11,65 +11,67 @@ Prerequisites
 The following is needed to connect a Nuts node to a network:
 
 1. A runnable node.
-2. The public address of one or more remote nodes you'd like to use as bootstrap nodes.
+2. A network you want to join.
 3. A TLS client- and server certificate which is accepted by the other nodes in the network (e.g. PKIoverheid).
-4. A truststore containing the CA trust anchors for TLS certificates the network you're connecting to accepts (e.g. PKIoverheid).
+4. The public address of one or more remote nodes you'd like to use as bootstrap nodes.
+5. A node Identity to identify yourself in te network
 
-To connect to the development network you can use the ``nuts-development-network-ca`` by cloning ``https://github.com/nuts-foundation/nuts-development-network-ca``.
-That project contains scripts to generate certificates, and a truststore.
+Networks
+========
 
-There are 2 networks for development/integration purposes:
-- `development`: where new features are tested. Nodes will generally run the newest (not yet released) version of the Nuts node.
-- `stable`: for integrating your software with Nuts and testing with other vendors. Nodes will generally run the latest released version (or at least a recent one).
+A network is a set of nodes who trust each other. They share a single root transaction. Trust is put in one or more network authorities who issue certificates (Certificate Authorities or CAs). Each node can be configured with a certificate and a set of trusted Certificate Authorities.
```

Comment:
It's trust chain right? I trust a root who endorses a node so I trust the node.

Reply:
Then, what kind of trust do we talk about here? What is derived from the trust?

What "trust" comes from a PKI CA is commonly known (since it's a established concept), here it is not. What does the sentence add? I'd say it only adds confusing because we also have vendors trust each other issuing specific VC types.